src/components/auth/register/RegisterForm.tsx

```diff
@@ -33,7 +34,10 @@ const validationSchema: yup.SchemaOf<RegisterFormData> = yup
     lastName: name.required(),
     email: email.required(),
     password: password.required(),
-    'confirm-password': yup.string().oneOf([yup.ref('password')], 'validation:password-match'),
+    'confirm-password': yup
```

Comment:
Let's use `confirmPassword` instead of kebab cased name. Just like `lastName`

Reply:
See 

https://github.com/podkrepi-bg/frontend/blob/df9850f26818e8a6b7c6125285920ace898e0d04/src/components/auth/changePassword/ChangePasswordForm.tsx#L29-L33

Reply:
@jozef-drabik Since we use this functionality in more than one place it's gonna be appropriate to extract it to the common validators at: https://github.com/podkrepi-bg/frontend/blob/8dc9da1b0ca7994dfef6f07ce2402f972fa8df83/src/common/form/validation.ts

Reply:
i used camelCase now for confirmPassword as well and functionality is extracted in form/validation.ts

Reply:
i used camelCase now for confirmPassword as well and functionality is extracted in form/validation.ts